README.md

```diff
@@ -164,3 +168,10 @@ l * incorrect_above + (1-l) * correct_below
 certain threshold and `correct_below` describes the fraction of correct 
 predictions below a certain threshold. Threshold and `l` can be configured 
 through environment variables.
+
+# Github Action
+
+`action.yml` defines an action that you can include in your CI/CD workflow. See `.github/workflows/nlu-hyperopt.yml` for an example of how to use this 
```

Comment:
I'd actually add the example pipeline directly to the Readme. From all the GitHub actions I have seen so far, this seems to be common practice.

Reply:
Ok that makes sense, I'll look at some examples of other actions.